src/Component/Config/src/Loader/FileLoader.js

```diff
@@ -93,6 +96,8 @@ class FileLoader extends Loader {
     }
 
     /**
+     * @private
```

Comment:
`@internal` implies public or protected

Reply:
so, what should I put in this one? For me, as it is used only in this class, this was private. Probably this should be protected then.

Reply:
Given that the method name starts with an underscore it's probably protected.